osquery/main/windows/main.cpp

```diff
@@ -39,24 +39,14 @@ static SERVICE_STATUS kServiceStatus = {0};
 static const std::string kStopEventName{"osqueryd-service-stop-event"};
 
 /// Logging for when we need to debug this service
-#define SLOG(...) ::osquery::DebugPrintf("[osqueryd] " __VA_ARGS__)
-
-void DebugPrintf(const char* fmt, ...) {
-  va_list vl;
-  va_start(vl, fmt);
-
-  int size = _vscprintf(fmt, vl);
-  if (size > 0) {
-    char* buf = static_cast<char*>(
-        ::HeapAlloc(GetProcessHeap(), HEAP_ZERO_MEMORY, size + 2));
-    if (buf != nullptr) {
-      _vsprintf_p(buf, size + 1, fmt, vl);
-      ::OutputDebugStringA(buf);
-      ::HeapFree(GetProcessHeap(), 0, buf);
-    }
-  }
+#define SLOG(s) ::osquery::DebugPrintf(s)
 
-  va_end(vl);
+void DebugPrintf(const std::string& s) {
+  auto dbgString = "[osqueryd] " + s;
+  if (isDebuggerPresent()) {
+    ::OutputDebugStringA(dbgString.c_str());
+  }
+  LOG(ERROR) << s;
```

Comment:
On second thought, not entirely sure this should be `LOG(ERROR)`, perhaps `LOG(INFO)` would be sufficient as we use this function for standard information messages also?